man/systemctl.xml

```diff
@@ -761,8 +761,9 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
 
           <listitem>
             <para>Restart one or more units specified on the command
-            line. If the units are not running yet, they will be
-            started.</para>
+            line. This is done by running a stop, and then a start 
+            on the unit. If the units are not running yet, they will 
```

Comment:
Do you mind to make this a little more compact and fluid to read, like "Stop and then start one or more units..". The "restart" word is already there right before, so it's not necessary to repeat it. Also, the documentation on `try-restart` should be updated in the same way IMHO.

Thanks!